Runtimes/Core/runtime/CMakeLists.txt

```diff
@@ -2,6 +2,16 @@ configure_file("CMakeConfig.h.in"
   "${PROJECT_BINARY_DIR}/include/swift/Runtime/CMakeConfig.h"
   ESCAPE_QUOTES @ONLY)
 
+install(FILES
```

Comment:
I tend to prefer to group all the install rules at the end of the file. That way the file flows logically: build rules for all the targets and then the rules to install the distributed components.